pages/1.10/installing/oss/cloud/gce/index.md

```diff
@@ -6,283 +6,241 @@ navigationTitle: GCE
+```
 
-### Configure Docker
+##### All Roles Simultaniously
+###### Not supported by dcos.io but it works without dcos_skip_checks enabled.
 
-1.  You must disable SELinux for Docker to work. Make the following change to `/etc/selinux/config`:
+```bash
+terraform apply -var-file desired_cluster_profile.tfvars -var state=upgrade
+```
 
-    ```bash
-    SELINUX=disabled
-    ```
 
-1.  Reboot host.
+## Maintenance
 
-1.  To install Docker add the Yum repo.
+If you would like to add more or remove (private) agents or public agents from your cluster, you can do so by telling terraform your desired state and it will make sure it gets you there. For example, if I have 2 private agents and 1 public agent in my `-var-file` I can always override that flag by specifying the `-var` flag. It has higher priority than the `-var-file`.
 
-    ```bash
-    sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
-    [dockerrepo]
-    name=Docker Repository
-    baseurl=https://yum.dockerproject.org/repo/main/centos/7/
-    enabled=1
-    gpgcheck=1
-    gpgkey=https://yum.dockerproject.org/gpg
-    EOF
-    ```
+### Adding Agents
 
-1.  Install the Docker package.
+```bash
+terraform apply \
+-var-file desired_cluster_profile.tfvars \
+-var num_of_private_agents=5 \
+-var num_of_public_agents=3
+```
 
-    ```bash
-    sudo yum install docker-engine-1.11.2
-    ```
+### Removing Agents
 
-1.  Add following changes to `/usr/lib/systemd/system/docker.service`.
+```bash
+terraform apply \
+-var-file desired_cluster_profile.tfvars \
+-var num_of_private_agents=1 \
+-var num_of_public_agents=1
+```
 
-    ```bash
-    ExecStart=/usr/bin/docker daemon --storage-driver=overlay
-    ```
+**Important**: Always remember to save your desired state in your `desired_cluster_profile.tfvars`
 
-1.  Reload systemd.
+## Redeploy an existing Master
 
-    ```bash
-    sudo systemctl daemon-reload
-    ```
+If you wanted to redeploy a problematic master (ie. storage filled up, not responsive, etc), you can tell terraform to redeploy during the next cycle.
 
-1.  Start Docker.
+**NOTE:** This only applies to DC/OS clusters that have set their `dcos_master_discovery` to `master_http_loadbalancer` and not `static`.
 
-    ```bash
-    sudo systemctl start docker.service
-    ```
+### Master Node
 
-1.  Verify that Docker works.
+#### Taint Master Node
 
-    ```bash
-    sudo docker run hello-world
-    ```
+```bash
+terraform taint google_compute_instance.master.0 # The number represents the agent in the list
+```
 
-## <a name="install"></a>Install the DC/OS GCE scripts
+#### Redeploy Master Node
 
-1.  Download the `dcos-gce` scripts
+```bash
+terraform apply -var-file desired_cluster_profile.tfvars
+```
 
-    ```bash
-    git clone https://github.com/dcos-labs/dcos-gce
-    ```
+## Redeploy an existing Agent
 
-1.  Change directory.
+If you wanted to redeploy a problematic agent, (ie. storage filled up, not responsive, etc), you can tell terraform to redeploy during the next cycle.
```

Comment:
Change:
"If you wanted to redeploy a problematic agent, (ie. storage filled up, not responsive, etc)"
to:
"If you want to redeploy a problematic agent (i.e., storage filled up, not responsive, etc.)"